src/server/apsystem.test.ts

```diff
@@ -133,6 +133,21 @@ test('getActor uses regular fetch when fromActor is not provided', async t => {
   }, 'getActor should use regular fetch when fromActor is not provided')
 })
 
+// Test for successful Webfinger fetch
+test('mentionToActor successfully fetches data from Webfinger', async t => {
```

Comment:
Mind modifying this test to also stub out the host-meta file?

Try storing the actual webmention at `/.well-known/webfinger/acct:test@domain.com` and the following for the host-meta file:

```
<?xml version="1.0" encoding="UTF-8"?>
<XRD xmlns="http://docs.oasis-open.org/ns/xri/xrd-1.0">
  <Link rel="lrdd" template="https://domain.com/.well-known/webfinge/{uri}"/>
</XRD>
```

This will help us know whether the parsing works for sure.